ui/urls.py

```diff
@@ -27,4 +28,5 @@
     url(r'^500/$', page_500, name='ui-500'),
     url(r'^learner/(?P<user>[-\w.]+)?/?', UsersView.as_view(), name='ui-users'),
     url(r'^{}$'.format(TERMS_OF_SERVICE_URL.lstrip("/")), terms_of_service, name='terms_of_service'),
+    url(r'^robots\.txt$', TemplateView.as_view(template_name='robots.txt', content_type='text/plain')),
```

Comment:
I would second this sentiment. There's no sense in having to go through Django to serve a text file.